subprojects/docs/src/docs/userguide/javaProjectMainLayout.adoc

```diff
@@ -11,11 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-| `src/main/java`
-| 
-| Production Java source
-
-| `src/main/resources`
-| 
-| Production resources
+`src/main/java`::
+Production Java source.
```

Comment:
Here and in a few other places we use singular "source" while we use plural 
for the Groovy/Scala ones.